### DISCUSSION TITLE
Planned features discussion

### DISCUSSION THREAD

**Discussion by Eldenroot** [mybbgroup/Bookmark-Post#9] [General]
I installed it and I am impressed!

I use bookmarks in Facebook, it works in a similar way and I think that many people are familiar with this system.

I see planned features list, I like adding notes idea. Also would be great to create user friendly list in UCP with more detailed info - thread, last activity, etc.

Good job.
Please leave this ticket opened for discussion.

**Comment by effone** [comment]
Updated planned feature list:

- Add personalized notes against each bookmarked post
- AJAX bookmark removal
- Remove bookmarks directly from UCP
- Show number of posts bookmarked by the user in user profile
- Show number of posts made by the user that others bookmarked in user profile
- Limit bookmarking per usergroup (might be useful for those paid group promotion sites)
- Export bookmarked posts (?)

Feel free to add more.

> Please leave this ticket opened for discussion.

We can open and make use of the `discussion` tab for this repo

**Comment by Eldenroot** [comment]
First three points are really valuable ideas which extend usability a lot.

**Comment by Eldenroot** [comment]
Also would be nice to see some kind of indicator (different color/icon) for threads where you have bookmarked posts.

I suggest a bookmark icon next to the attachment icon.

Edit: also add a possibility (maybe css class) for bookmarked post background to use any other different color to easily recognize them.
